MsCorePkg/Library/MemoryProtectionExceptionHandlerLib/MemoryProtectionExceptionHandlerLib.c

```diff
@@ -24,7 +24,8 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include <Library/MsWheaEarlyStorageLib.h>
 #include <Library/PeCoffGetEntryPointLib.h>
 
-#define IA32_PF_EC_ID  BIT4
+#define IA32_PF_EC_ID        BIT4
+#define EXCEPT_STACK_COOKIE  0x40

```

Comment:
It would be kind of nice if a globally visible FixedAtBuild PCD could set the interrupt value used everywhere. Did you consider that?

Reply:
I didn't consider that, but that seems like a potentially confusing design unless the stack cookie code also uses the fixed at build PCD when generating the interrupt. I suppose we can hook the PCD value into the stack cookie check code if there aren't any restrictions on the execution environment.